### ISSUE TITLE
ModuleNotFoundError: No module named 'scenic.train_lib_deprecated'

### ISSUE THREAD

**Issue by molu-ggg** [google-research/scenic#1140]
I want to use vivit , 
python==3.10, scenic==0.0.1 ,
but train_lib_deprecated not in scenic. 
Do you have a solution? Or which version of Scenic should be used?

**Comment by ffffffly923** [comment]
+1

**Comment by pepper725** [comment]
+1

**Comment by pepper725** [comment]
Did you solve the problem?

**Comment by dddcyy** [comment]
just replace scenic,train_lib_depreacated with train_lib

**Comment by ffffffly923** [comment]
can u train the  model correctly?

**Comment by dddcyy** [comment]
no, I can't. There are always many problems if I want to run vivit. I am still fixing this horrible thing[sadly]

**Comment by tj-messi** [comment]
@dddcyy  can we have a talk?

**Comment by zmasih** [comment]
I faced the same issue. Any updates?

**Comment by dddcyy** [comment]
> [@dddcyy](https://github.com/dddcyy) can we have a talk?

Well, I gave up
